src/Microsoft.Health.Dicom.Functions.Abstractions/Migration/BlobMigrationInput.cs

```diff
@@ -3,9 +3,13 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
+using System;
+
 namespace Microsoft.Health.Dicom.Functions.Migration;
 
 public class BlobMigrationInput
 {
     public BatchingOptions Batching { get; set; }
+
+    public DateTime FilterTimeStamp { get; set; }
```

Comment:
Should this just be nullable?

Reply:
Yes, it can be. I was checking for DateTime.MinValue instead of null. But making it explicit null makes more sense